pkg/cloudprovider/providers/oci/load_balancer.go

```diff
@@ -521,7 +539,6 @@ func (cp *CloudProvider) EnsureLoadBalancer(ctx context.Context, clusterName str
 		dimensionsMap[metrics.ComponentDimension] = lbMetricDimension
 		dimensionsMap[metrics.ResourceOCIDDimension] = lbName
 		metrics.SendMetricData(cp.metricPusher, getMetric(loadBalancerType, Update), time.Since(startTime).Seconds(), dimensionsMap)
-		return nil, err
```

Comment:
return needs to remain as it is also there in internal

Reply:
Added the return statement here.